### PR TITLE
fix(external-video): Viewers can now mute/unmute their view of external video

### PR DESCRIPTION
**Pull request by gabriellpr**
The viewer can't mute the volume. 
https://user-images.githubusercontent.com/19312495/160689998-3fdaa0f9-b9d9-4619-b009-ecd60c7be966.mp4

Now, the viewer can mute and unmute the volume with no problems.
https://user-images.githubusercontent.com/19312495/160690014-36a233eb-64f5-4d68-b06a-b43adc0e14d4.mp4

### CODE REVIEW

**Comment by PhMemmel** [comment]
Probably closes #14618?

File: bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx [hunk 1]
```diff
@@ -378,10 +378,10 @@ class VideoPlayer extends Component {
   }
 
   getMuted() {
-    const { mutedByEchoTest } = this.state;
+    const { mutedByEchoTest, muted } = this.state;
     const intPlayer = this.player && this.player.getInternalPlayer();
 
-    return intPlayer && intPlayer.isMuted && intPlayer.isMuted() && !mutedByEchoTest;
+    return (intPlayer && intPlayer?.isMuted && intPlayer?.isMuted?.() && !mutedByEchoTest) || muted;
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/external-video-player/component.jsx:384]: I think if we check `intPlayer &&` we don't need to do `intPlayer?` in the rest of the statement...

> isMuted?.()

I've never seen optional chaining used for a function in our code before :smile_cat: I had to look it up but apparently it's a thing! https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

**Review by JoVictorNunes — APPROVED** [review]
Works as expected!

**Comment by antobinary** [comment]
Nice work! :)

**Pull request merged**